### PR TITLE
Adding https capability to the webhook webserver

### PR DESCRIPTION
**Pull request by mreiger**

### CODE REVIEW

File: pkg/controllers/audit/webhook.go [hunk 1]
```diff
@@ -26,10 +27,22 @@ func NewController(logger *zap.SugaredLogger, client hec.HEC) *Controller {
 
 // AuditEvent handles an audit event
 func (c *Controller) AuditEvent(response http.ResponseWriter, request *http.Request) {
-	c.logger.Infow("received audit event", "request", request.Body)
-
-	event := hec.NewEvent("event one")
-	event.SetTime(time.Now())
+	Body, _ := ioutil.ReadAll(request.Body)
```
Comment (Gerrit91) [inline pkg/controllers/audit/webhook.go]: It confuses me that you started this variable with a capital letter. I would expect that only "public" variables should start with a capital letter while function scope variables should start with small letters. However, I cannot find a word about using capital letters in function scope variables in any Golang style guide and golang linters and IDEs seem fine with this. Mh.

**Review by Gerrit91 — APPROVED** [review]
Apart from the one comment, it looks good to me.

File: pkg/controllers/audit/webhook.go (hunk 1 above)
Comment (mreiger) [reply pkg/controllers/audit/webhook.go]: Makes sense anyway - changed as you suggested.

Comment (Gerrit91) [reply pkg/controllers/audit/webhook.go]: Thanks. I maybe I find evidence one day. :laughing:

**Pull request merged**